schema/accepted_schema.sql

```diff
@@ -0,0 +1,910 @@
+    _id varchar(27) PRIMARY KEY,
+    "userId" varchar(27),
+    "documentId" text,
+    "documentType" text,
+    "extraData" jsonb,
+    "link" text,
+    "title" text,
+    "message" text,
+    "type" text,
+    "deleted" bool DEFAULT false,
+    "viewed" bool DEFAULT false,
+    "emailed" bool DEFAULT false,
+    "waitingForBatch" bool DEFAULT false,
+    "schemaVersion" real DEFAULT 1,
+    "createdAt" timestamptz)
+
+-- Schema for "PetrovDayLaunchs", hash: a8916451920164821ceecfc5f28e77a3
+CREATE TABLE "PetrovDayLaunchs" (
+    _id varchar(27) PRIMARY KEY,
+    "launchCode" text,
+    "hashedLaunchCode" text,
+    "userId" text,
+    "schemaVersion" real DEFAULT 1,
+    "createdAt" timestamptz)
+
+-- Schema for "PodcastEpisodes", hash: 2703ae9430d3a2feaabc0bbafaa81c11
+CREATE TABLE "PodcastEpisodes" (
+    _id varchar(27) PRIMARY KEY,
+    "podcastId" varchar(27),
+    "title" text NOT NULL,
+    "episodeLink" text NOT NULL,
+    "externalEpisodeId" text NOT NULL,
+    "schemaVersion" real DEFAULT 1,
+    "createdAt" timestamptz)
+
+-- Schema for "Podcasts", hash: ae52c5b1d8aaf75526fe3fd7c4f95480
+CREATE TABLE "Podcasts" (
+    _id varchar(27) PRIMARY KEY,
+    "title" text NOT NULL,
+    "applePodcastLink" text,
+    "spotifyPodcastLink" text,
+    "schemaVersion" real DEFAULT 1,
+    "createdAt" timestamptz)
+
+-- Schema for "PostRelations", hash: 750051194d0eaaa745f8776e4f67b8f4
+CREATE TABLE "PostRelations" (
+    _id varchar(27) PRIMARY KEY,
+    "type" text,
+    "sourcePostId" varchar(27),
+    "targetPostId" varchar(27),
+    "order" real,
+    "schemaVersion" real DEFAULT 1,
+    "createdAt" timestamptz)
+
+-- Schema for "Posts", hash: 22c85503ac37424e9b7037a1be34a60b
+CREATE TABLE "Posts" (
+    _id varchar(27) PRIMARY KEY,
+    "postedAt" timestamptz,
+    "modifiedAt" timestamptz,
+    "url" varchar(500),
+    "title" varchar(500) NOT NULL,
+    "slug" text,
+    "viewCount" real DEFAULT 0,
+    "lastCommentedAt" timestamptz,
+    "clickCount" real DEFAULT 0,
+    "deletedDraft" bool DEFAULT false,
+    "status" real,
+    "isFuture" bool,
+    "sticky" bool DEFAULT false,
+    "stickyPriority" integer DEFAULT 2,
+    "userIP" text,
+    "userAgent" text,
+    "referrer" text,
+    "author" text,
+    "userId" varchar(27),
+    "question" bool DEFAULT false,
+    "authorIsUnreviewed" bool DEFAULT false,
+    "readTimeMinutesOverride" real,
+    "submitToFrontpage" bool DEFAULT true,
+    "hiddenRelatedQuestion" bool DEFAULT false,
+    "originalPostRelationSourceId" text,
+    "shortform" bool DEFAULT false,
+    "canonicalSource" text,
+    "nominationCount2018" real DEFAULT 0,
+    "nominationCount2019" real DEFAULT 0,
+    "reviewCount2018" real DEFAULT 0,
+    "reviewCount2019" real DEFAULT 0,
+    "reviewCount" real DEFAULT 0,
+    "reviewVoteCount" real DEFAULT 0,
+    "positiveReviewVoteCount" real DEFAULT 0,
+    "reviewVoteScoreAF" real DEFAULT 0,
+    "reviewVotesAF" real[] DEFAULT '{}' ::real[][],
```

Comment:
Should be:
```suggestion
    "reviewVotesAF" real[] DEFAULT '{}' ::real[],
```
There is a bug in the default generating code, which I've fixed in this PR: https://github.com/ForumMagnum/ForumMagnum/pull/5926